libs/blocks/tabs/tabs.css

```diff
@@ -548,6 +625,10 @@
     padding-inline: 0;
     min-block-size: 56px;
   }
+
+  .tabs.segmented-control .tabList button {
+    padding: 10px 17.5px;
```

Comment:
Sub-pixel values can lead to rendering inconsistencies, this should ideally be rounded.